solidity/utils/README.md

```diff
@@ -25,23 +25,23 @@ Here is an example of the initial configuration file which should be provided to
 {
     "reserves": [
         {
-            "type": 0,
             "symbol": "XXX",
+            "decimals": 18,

```

Comment:
we probably shouldn't have the decimals in the config if the user can't change it... we should just deploy every token with 18 decimals and query the blockchain in the script to tell the decimals of each token.

Reply:
he can

Reply:
gotcha